examples/ex19p.cpp

```diff
@@ -348,8 +348,6 @@ int main(int argc, char *argv[])
    delete pmesh;
 
    MPI_Finalize();
-
-   return 0;
```

Comment:
Why not?

Reply:
I cannot seem to reproduce the behavior I saw last night.  I'll put these back and run more tests.